pulpcore/cli/ostree/repository.py

```diff
@@ -81,6 +87,33 @@ def repository(ctx: click.Context, pulp_ctx: PulpContext, repo_type: str) -> Non
 repository.add_command(label_command(decorators=nested_lookup_options))
 
 
+def ref_name_callback(ctx: click.Context, param: click.Parameter, value: str) -> str:
+    if value:
+        pulp_ctx = ctx.find_object(PulpContext)
+        assert pulp_ctx is not None
+        ctx.obj = PulpOstreeRefContentContext(pulp_ctx, entity={"name": value})
```

Comment:
I think, "name" alone is not enough to identify a ref uniquely in pulp, no?

Reply:
Yep, you are right. The checksum of a referenced commit should be specified to make it unique. I am opening a new PR with the fix in no time.